README.md

```diff
@@ -1,11 +1,26 @@
 # LearnHub.mk App
-The frontend app for LearnHub.mk
+- LearnHub.mk is an online learning platform designed to provide comprehensive educational resources across various subjects. Our mission is to make learning accessible to everyone, everywhere. This repository contains the frontend application for LearnHub.mk, built with a focus on user experience and performance.
+# Features
+- Access a wide range of educational materials.
+- Community forums: Participate in discussions, share insights, and seek support from a vibrant community of learners and professionals.
+- Collaborative Projects: Join or initiate projects within the community, applying your skills in real-world scenarios and collaborating with others.
+- Mentorship Opportunities: Connect with industry experts and mentors who can provide guidance, support, and valuable feedback on your projects or career path.
 # Technology used: 
-- Next.js
-- React Query
-- SASS
-- React Bootstrap
-- ESLint
-# Setup Project
-- npm install 
-- npm run dev
+- Next.js:  Chosen for its server-side rendering capabilities, which improve SEO and performance.
+- React Query: Utilizes for efficient data fetching, caching, and state management in React.
+- SASS: Used for writing more maintainable CSS with support for variables, nesting, and mixins.
+- React Bootstrap: Provides a suite of reusable components for building a responsive UI quickly.
+- ESLint: Ensures code quality and consistency across the project.
+# Setup and Installation
+Before starting, ensure you have Node.js installed on your machine.
+1. Clone the repository to your local machine.
+2. Navigate to the project directory and run `npm install` to install dependencies.
+3. Start the development server with `npm run dev`.
+4. Open `http://localhost:3000` in your browser to view the app.
```

Comment:
http://localhost:3000 should be  a hyperlink, instead of a code block.
Use this instead: [localhost](http://localhost:3000) ``[localhost](http://localhost:3000)``

Reply:
I think that now it is solved, look at the readme again.